recipes-ni/ni-systemreplication/files/nisystemreplication

```diff
@@ -100,15 +106,27 @@ image_get () {
 
 # ---------------------- Set Image ----------------------
 
-# Presents a list of existing images and let's user select one
+# Presents a list of existing compatible images and let's user select one
 select_image_from_list () {
 	[ ! -d $NIRECOVERY_MOUNTPOINT/Images ] && return
 
-	readarray -t images <<< "$(find $NIRECOVERY_MOUNTPOINT/Images -name systemimage.tgz | sort | sed "s,$NIRECOVERY_MOUNTPOINT/Images/,," | sed 's,/systemimage.tgz,,')"
+	images=()
+
+	for image in $(ls $NIRECOVERY_MOUNTPOINT/Images); do
+		if [ ! -f $NIRECOVERY_MOUNTPOINT/Images/$image/systemimage.tgz ] || [ ! -f $NIRECOVERY_MOUNTPOINT/Images/$image/systemimage.conf ]; then
```

Comment:
Can we warn when this condition happens? I imagine that it will be unexpected for the user.

Reply:
Done.